pkg/operator/periodicbackupcontroller/periodicbackupcontroller.go

```diff
@@ -81,6 +87,12 @@ func (c *PeriodicBackupController) sync(ctx context.Context, _ factory.SyncConte
 	}
 
 	for _, item := range backups.Items {
```

Comment:
sorry I totally forgot, we need a case for when the default was removed
```suggestion
        defaultFound := false
	for _, item := range backups.Items {
```

Reply:
and below the loop (can't suggest) you need to set it to nil:

```
if !defaultFound {
   c.backupVarGetter.SetBackupSpec(nil)
}
```

Reply:
gotcha 👍🏽